tests/test_hsm.py

```diff
@@ -405,7 +405,31 @@ def runMeasurement(self, algorithmName, imageid, x, y, v):
         source.setFootprint(afwDetection.Footprint(afwGeom.SpanSet(exposure.getBBox(afwImage.PARENT))))
         plugin.measure(source, exposure)
 
-        return source
+        # Get the trace radius of the PSF and GalSim images to use in the
+        # EstimateShear call.
+        psfSigma = exposure.getPsf().computeShape(center).getTraceRadius()
+        bbox = source.getFootprint().getBBox()
+        bounds = galsim.bounds.BoundsI(bbox.getMinX(), bbox.getMaxX(), bbox.getMinY(), bbox.getMaxY())
+        image = galsim.Image(exposure.image[bbox].array, bounds=bounds, copy=False)
+        psf = galsim.Image(psfImg.array, copy=False)
+
+        # Retrieve the measurement type that Galsim outputs after estimation.
+        # NOTE: not passing weight, badpix, and sky_var, as the objective here
+        # is solely to deduce the meas_type for this setup.
+        postEstimationMeasType = galsim.hsm.EstimateShear(
+            gal_image=image,
+            PSF_image=psf,
+            shear_est=control.shearType,
+            recompute_flux="FIT",
+            guess_sig_gal=2.5 * psfSigma,
+            guess_sig_PSF=psfSigma,
+            precision=1.0e-6,
+            guess_centroid=galsim.PositionD(center.getX(), center.getY()),
+            strict=True,
+            hsmparams=None,
+        ).meas_type
```

Comment:
Should we make it even more minimal?

Reply:
What do you have in mind here? Unless the values passed here are the default values for `EstimateShear`, I'd be a bit skeptical about it.

Reply:
This is regarding your comment [here](https://github.com/lsst/meas_extensions_shapeHSM/pull/47#discussion_r1413199861), where you noted that it is not necessary to verify if the `meas_type` output by GalSim aligns with our assumptions for each measurement within the plugin. You suggested we validate this in one of the unit tests. The ones that are not passed will be set to the default of `EstimateShear`. We are not passing `badpix` and `sky_var` because obtaining them here is unnecessary, as they do not affect the value we aim to extract, which is simply `mea_stype`.

Reply:
From GalSim:
``` python
def EstimateShear(gal_image, PSF_image, weight=None, badpix=None, sky_var=0.0,
                  shear_est="REGAUSS", recompute_flux="FIT", guess_sig_gal=5.0,
                  guess_sig_PSF=3.0, precision=1.0e-6, guess_centroid=None,
                  strict=True, hsmparams=None):
                  ...
```

Reply:
So, we can do:
``` python
        postEstimationMeasType = galsim.hsm.EstimateShear(
            gal_image=image,
            PSF_image=psf,
            shear_est=control.shearType,
            guess_sig_gal=2.5 * psfSigma,
            guess_sig_PSF=psfSigma,
            guess_centroid=galsim.PositionD(center.getX(), center.getY()),
        ).meas_type
```

Reply:
Yeah, you could remove all the guess parameters and pass `strict=False`. The `meas_type` should be the same regardless of the guesses.

Reply:
Sounds good, except I still need to pass `guess_centroid`, otherwise it returns `None` for `meas_type` with `strict=False` and throws this error with `strict=True`:
```
galsim.errors.GalSimHSMError: Bounds don't make sense
````